src/components/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,33 @@
+import React,{ useState , useContext ,useEffect } from "react";
+import { getMonth } from "./CalenderComponent/util";
+import Month from "./CalenderComponent/Month";
+import GlobalContext from "./context/GlobalContext";
+import CalendarHeader from "./CalenderComponent/CalendarHeader";
+import calenderSidebar from "./CalenderComponent/calenderSidebar";
```

Comment:
where is this component? and is it being used in here?

Reply:
The Calendar.tsx is the final Calendar component which has been imported to \src\app\events\page.tsx for the purpose of displaying it on the web app.

Reply:
talking about `import calenderSidebar from "./CalenderComponent/calenderSidebar";`

Reply:
Ok got it, yes its an unnecessary component which was used before , I will fix it.